webapp/channels/src/sass/base/_css_variables.scss

```diff
@@ -29,9 +29,6 @@
     --sidebar-unread-text: #fff;
     --sidebar-team-background: #0b428c;
     --secondary-blue: #22406d;
-    --denim-button-bg: #1c58d9;
```

Comment:
Looking at the plugins I have checked out locally, this one is still used in Calls and the User Survey plugin, so we're going to have to keep it around for a bit until we can remove its usage from them as well.

For the time being, you should probably re-add it, and then I think we might be able to prevent new web app code from using it by updating `channels/.stylelintrc.json` to add [this rule to it](https://stylelint.io/user-guide/rules/declaration-property-value-disallowed-list/) to specify that `color`s, `border-color`, and `background-color`s can't be set to `var(--denim-button-bg)`. Let me know if you could use any help with that

Reply:
@hmhealey have a look at the latest commit and let me know what you think

Reply:
That works in most cases, but I think I found a few that it wouldn't catch. I think I know how to fix them though

Reply:
Thanks for the update. I committed your suggestion.